### PR TITLE
Add current versions section to main page

### PR DESCRIPTION
**Pull request by lw64**

### CODE REVIEW

**Comment by lw64** [comment]
Fixes #152 (partially)

**Comment by lw64** [comment]
grafik

**Comment by lw64** [comment]
grafik

**Comment by lw64** [comment]
@colinkiama can the "stable" entry be removed whenever "stable" and "stable lts" are equal?

**Comment by colinkiama** [comment]
Yes, conditional rendering is supported.

See: https://keats.github.io/tera/docs/#if

**Comment by lw64** [comment]
Ok, itsconditional now.

**Comment by lw64** [comment]
Any improvements for style and position on the main page appreciated :)

File: templates/partials/versions.html [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<h2>{{ trans(key='current_versions', lang=lang) }}</h2>
+{{ trans(key='current_versions_text', lang=lang) }}
```
Comment (colinkiama) [inline templates/partials/versions.html:2]:
```suggestion
```
I think that this line is redundant.

We know that these are the current versions of Vala available from the title of the section.

File: templates/partials/versions.html [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+<div>
```
Comment (colinkiama) [inline templates/partials/versions.html:3]:
```suggestion
```
Doesn't seem to serve any purpose

File: templates/partials/versions.html [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    </li>
+    {% endif %}
+    <li>
+      <a href="https://gitlab.gnome.org/GNOME/vala/-/blob/{{ config.extra.versions.lts }}/NEWS">Vala {{ config.extra.versions.lts }}</a> ({{ trans(key='stable_lts', lang=lang) }})
+    </li>
+    <li>
+      <a href="https://gitlab.gnome.org/GNOME/vala/-/commits/main">Vala {{ config.extra.versions.devel }}</a> ({{ trans(key='devel', lang=lang) }})
+    </li>
+  </ul>
+</div>
```
Comment (colinkiama) [inline templates/partials/versions.html:17]:
```suggestion
```
Doesn't seem to serve any purpose

File: templates/partials/versions.html (hunk 1 above)
Comment (colinkiama) [reply templates/partials/versions.html:2]: If you insist on having the line there anyway, wrap it with the `<p>` tag.

**Comment by colinkiama** [comment]
Awesome, I'm happy with this!

**Comment by lw64** [comment]
nice and thank you!

**Pull request merged**